body-tracking-samples/simple_3d_viewer/main.cpp

```diff
@@ -268,5 +420,25 @@ int main(int argc, char** argv)
     k4a_device_stop_cameras(device);
     k4a_device_close(device);
 
+
+}
+
+int main(int argc, char** argv)
+{
+    InputSettings inputSettings;
+    PrintUsage();
+    ParseInputSettingsFromArg(argc, argv, inputSettings);
```

Comment:
Please check for a failure

Reply:
I have done this, please see new commit.